app/src/main/java/com/jerboa/ui/components/settings/lookandfeel/LookAndFeelActivity.kt

```diff
@@ -107,6 +107,8 @@ fun LookAndFeelActivity(
     val markAsReadOnScroll = rememberBooleanSettingState(settings.markAsReadOnScroll)
     val autoPlayGifs = rememberBooleanSettingState(settings.autoPlayGifs)
 
+    val swipeToActionPreset = rememberIntSettingState(settings.swipeToActionPreset)
```

Comment:
Did you remove the actual settings button to change this preset by accident? Either way it needs to exist

Reply:
Yep. Fixed it.